docs/integration/mining.md

```diff
@@ -59,7 +60,20 @@ Once the mining pool receives a valid `nonce` from the miner, it can send the bl
 
 Then, refer to the code [here](https://github.com/alephium/mining-pool/blob/master/lib/daemon.js#L49) to construct a valid `SubmitBlock` message and send it to the full node.
 
-After the full node verifies the block, it will send a `SubmitBlockResult` message to inform the mining pool whether the block is valid. Refer to the code [here](https://github.com/alephium/mining-pool/blob/master/lib/messages.js#L72) to parse the `SubmitBlockResult` message.
+After the full node verifies the block, it will send a `SubmitBlockResult` message to inform the mining pool whether the block is valid. Refer to the code [here](https://github.com/alephium/mining-pool/blob/master/lib/messages.js#L77) to parse the `SubmitBlockResult` message.
+
+### Migrate to the Latest Mining Protocol
+
+In the latest full node, the mining protocol introduces some breaking changes:
+
+1. Each message type includes a version number. The current mining protocol version is `1`.
+2. The `Job` message includes a block height field.
+3. The `SubmitResult` message includes a block hash field.
+
+To migrate to the latest full node:
+
+* For mining pool developers, please refer to the changes in [this commit](https://github.com/alephium/mining-pool/pull/66/commits/eacc46188b2f34245d510e59d1bf1e9f256ec611).
+* For miner developers, please refer to the changes in [this commit](https://github.com/alephium/gpu-miner/pull/61/commits/1715dbf3fe7115abadfce1eae51651637dc243c1).
```

Comment:
Miners are built based on pool protocol, they should not be impacted by the changes in mining protocols.